### PR TITLE
Restructured Contents Section 3

### PR DESCRIPTION
**Pull request by robertholl**
Reworded section 3.2 and added SFTP to go along with HTTPS.

### CODE REVIEW

**Comment by marcelocarlosbr** [comment]
Wouldn't it be better if SSL/TLS was discussed here instead of HTTPS/SFTP? Or even both, starting with SSL then moving on to HTTPS/SFTP/etc?

**Comment by robertholl** [comment]
That's a good thought.  I will update to reflect your suggestion.

**Comment by robertholl** [comment]
@marcelocarlosbr Thanks.  I made the suggested change and hope this is more what you had in mind.

**Comment by marcelocarlosbr** [comment]
Yes, sounds good. And we can always adjust it later once we have more contents in there :)

File: README.md [hunk 1]
```diff
@@ -14,7 +14,10 @@ Our detailed explanations should help the first type while we hope our checklist
 
 1. [The Security Checklist](security-checklist.md)
 2. What can go wrong?
-3. Securely transporting stuff: HTTPS explained
+3. Secure data transmission  
+3.1 SSL/TLS  
+3.2 HTTPS  
+3.3 SFTP
```
Comment (pdauu) [inline README.md]: 1

**Comment by robertholl** [comment]
This is my first time creating a merge request, so I am not certain how the entire process works or if there is something else I need to do.  Thanks.

**Comment by abhishek-anand** [comment]
@robertholl Thanks for this. We will be merging this along with some other table of contents restructuring we are planning.

**Comment by chadfurman** [comment]
I feel there should be mention of secure SSL algorithms and how to check for insecure algorithms.

i.e. https://www.owasp.org/index.php/Testing_for_Weak_SSL/TLS_Ciphers,_Insufficient_Transport_Layer_Protection_(OTG-CRYPST-001)#Example_3._Checking_for_Certificate_information.2C_Weak_Ciphers_and_SSLv2_via_nmap

**Pull request closed without merging**